.vscode/settings.json

```diff
@@ -23,11 +23,11 @@
           "keyword.control.flow.js", // await
           "keyword.control.conditional.js", // if
           "keyword.control.loop.js", // for
-          "keyword.operator.new.js", // new
+          "keyword.operator.new.js" // new
         ],
         "settings": {
-          "fontStyle": "italic",
-        },
```

Comment:
We are always adding the tailing `,` in every line.

Does prettier support the above rule?